src/scss/login.module.scss

```
@@ -98,32 +103,37 @@ $white: #fff;
   font-size: 1rem;
   transform: rotate(-90deg) translateY(50%);
   float: left;
-  margin-left: -4rem;
-  position: relative;
-  top: 45%;
+  margin-top: 10rem;
+  margin-left: -8rem;
   letter-spacing: 0.2rem;
 }
+@media only screen and (max-width: 561px) {
+  .coords {
+    font-size: 0.6rem;
+  }
+}
 
 .ellipses-container {
-  width: 35rem;
-  height: 35rem;
+  width: 20vw;
+  height: 20vw;
   border-radius: 50%;
   margin: 0 auto;
   position: relative;
   top: 1.5rem;
```

Could you change this to 1rem?

```suggestion
  top: 1rem;
```